components/navbar/navbar.scss

```diff
@@ -32,7 +32,7 @@
     width: 100%;
     height: 100%;
     background: rgba(0, 0, 0, 0.9);
-    z-index: 1000000;
+    z-index: map-get($z-indexes, nav-link-list);
```

Comment:
language nit: the plural of index is indices

Reply:
i looked it up online before i named this and it seemed like both were acceptable??? Anyways I'll rename it to indices.